presto-verifier/src/main/java/com/facebook/presto/verifier/rewrite/VerificationQueryRewriterFactory.java

```diff
@@ -15,42 +15,54 @@
 
 import com.facebook.presto.spi.type.TypeManager;
 import com.facebook.presto.sql.parser.SqlParser;
+import com.facebook.presto.sql.tree.BooleanLiteral;
+import com.facebook.presto.sql.tree.DoubleLiteral;
+import com.facebook.presto.sql.tree.Identifier;
+import com.facebook.presto.sql.tree.LongLiteral;
 import com.facebook.presto.sql.tree.Property;
+import com.facebook.presto.sql.tree.QualifiedName;
+import com.facebook.presto.sql.tree.StringLiteral;
 import com.facebook.presto.verifier.annotation.ForControl;
 import com.facebook.presto.verifier.annotation.ForTest;
 import com.facebook.presto.verifier.prestoaction.PrestoAction;
+import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableMap;
 
 import javax.inject.Inject;
 
 import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
 
+import static com.facebook.presto.sql.tree.BooleanLiteral.FALSE_LITERAL;
 import static com.facebook.presto.verifier.framework.ClusterType.CONTROL;
 import static com.facebook.presto.verifier.framework.ClusterType.TEST;
+import static java.lang.String.format;
 import static java.util.Objects.requireNonNull;
 
 public class VerificationQueryRewriterFactory
         implements QueryRewriterFactory
 {
     private final SqlParser sqlParser;
     private final TypeManager typeManager;
-    private final List<Property> tablePropertyOverrides;
-    private final QueryRewriteConfig controlConfig;
-    private final QueryRewriteConfig testConfig;
+    private final QualifiedName controlTablePrefix;
```

Comment:
I'm confused about the changes here as I lacked a lot of context. Why did we remove `QueryRewriteConfig` and use prefix?

Reply:
- `QueryRewriteConfig` used to have only 1 field: `tablePrefix`. Instead of storing the `QueryRewriteConfig` object, the change here is to extract and store that field directly.
- A new field `tableProperties` is added to `QueryRewriteConfig` in this PR. In the constructor, we're converting it from `Map<String, Object>` to `List<Property>` and store it list. If these is any Exception thrown in `constructProperties`, I want it to throw in the constructor so that injection will fail and verification will not start. Otherwise, if the Exception is thrown in the `create` method, verification is already started and all verifications will complete but with `FAILED` status.